crawlers/currency_price_tw.py

```diff
@@ -54,6 +64,9 @@ def crawl_data(self, instrument):
                 if date == self.last_modified_date[instrument]:
                     synchronized = True
                     break
+                if not (is_float(grids[3].text) and is_float(grids[4].text)):
+                    history.append((date, None, None))
```

Comment:
What if one of them is available?
Maybe we can let them be
`buy = float(...) if is_float(...) else None`
Or we can even modify is_float to be something like safe_float, which returns None if input is illegal.